CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 0.18.7
+
+* Smithy4s specific logic to extract manifest from jars can run on files that are not jars. Fixed in https://github.com/disneystreaming/smithy4s/pull/1351
```

Comment:
Describe the new fix, not the bug. 

"Fixes a bug that would occur jar-specific logic to be attempted on non-jar files "